src/components/RepoSubmission.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC, useState } from 'react'
+
+
+const RepoSubmission: FC = ()  => {
+  
+  const [buttonPlaceHolder, setButtonPlaceHolder] = useState<string>("Submit repo?")
+  const [isFormOpen, setIsFormOpen] = useState<boolean>(false)
```

Comment:
most of the types can be inferred here, should clean this up as much as possible

Reply:
I have no clue what's wrong with it if I explicitly set types rather than allowing the compiler to infer. Anyway, I refactored the code as per your direction, good sir! @0-vortex